src/styles/blocks/card.scss

```diff
@@ -0,0 +1,109 @@
+.card {
+  height: 408px;
+  width: 200px;
+  box-sizing: border-box;
+  border: 1px solid #f3f3f3;
+  border-radius: 5px;
+  padding: 0 16px;
+  transition: transform 300ms;
+
+  &__image {
+    display: block;
+    height: 134px;
+    width: 160px;
+    margin: 32px auto 0;
+  }
+
+  &__title {
+    height: 36px;
+    width: 166px;
+    font-family: inherit;
+    font-weight: 500;
+    line-height: 18px;
+    font-size: 12px;
+    color: $main_accent;
+    margin: 40px auto 0;
+  }
+
+  &:hover {
+    transform: scale(1.25);
+    .card__title {
+      color: #34568b;
+    }
+  }
+
+  &__code {
+    font-family: inherit;
+    font-weight: 400;
+    line-height: 14px;
+    font-size: 10px;
+    color: $secondary;
+    margin: 4px 0 0;
+  }
+
+  &__reviews {
+    display: flex;
+    justify-content: space-between;
+    align-items: flex-end;
+    margin: 16px 0 0;
+
+    &-text {
+      font-family: inherit;
+      font-weight: 400;
+      line-height: 14px;
+      font-size: 10px;
+      color: $main_accent;
```

Comment:
Variables for colors should be named semantically, for example, '$text-color', to improve code readability and maintainability.